taf/src/main/java/com/epam/tat/test/Constants.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.tat.test;
+
+public class Constants {
+
+    public static final int VISIBILITY_TIMEOUT_SECONDS = 15;
+    public static final int VALUE_TO_BE_TIMEOUT_SECONDS = 30;
+    public static final String LOGIN = "taf.alexander.gritsok";
+    public static final String PASSWORD = "WebDriverGo";
+    public static final String INCORRECT_VALUE = "qazxcdews11111111111";
```

Comment:
It's a good approach to randomize such data as it increases the probability to catch a bug. I may advise a RandomStringUtils.randomAlphanumeric() from Apache Commons for this purpose.
https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/RandomStringUtils.html#randomAlphanumeric(int)